client/noshuff.lua

```diff
@@ -1,40 +1,34 @@
 local disableShuffle = true
-local vehicle = nil
 
-RegisterNetEvent('baseevents:enteredVehicle', function (veh)
-    vehicle = veh
-    local ped = PlayerPedId()
-    while vehicle do
+AddEventHandler('ox_lib:cache:vehicle', function(_)

```

Comment:
1. Use lib.onCache, looks cleaner
2. Use the value that it returns instead of always using cache.vehicle
3. That underscore is pointless

Reply:
Updated